week-3/quote-generator/quotes.js

```diff
@@ -1,3 +1,45 @@
+
+document.addEventListener("DOMContentLoaded", function () {
+  const quoteElement = document.getElementById("quote");
+  const authorElement = document.getElementById("author");
+  const newQuoteButton = document.getElementById("new-quote");
+  const autoplayToggle = document.getElementById("autoplay-toggle");
+  const autoplayStatus = document.getElementById("autoplay-status");
```

Comment:
Fantastic job grouping the elements together! It's a great practice to organise and select all the necessary elements at the beginning. well done!